packages/oha/project.bri

```diff
@@ -0,0 +1,24 @@
+import * as std from "std";
+import { cargoBuild } from "rust";
+
+export const project = {
+  name: "dust",
```

Comment:
Typo

```suggestion
  name: "oha",
```

Reply:
Aie 😶‍🌫️ my bad, it was a copy and paste... Definitively, a lint rule could help to mitigate these issues. Does the LSP expose any lints? I didn't take a look yet.

Reply:
Ok, I see: `brioche check` being called in one of a CI job.

Reply:
Yeah, currently there's lints from ESLint and typescript-eslint, but there aren't any Brioche-specific lint rules in the LSP yet. I went with ESLint because it has a pretty good plugin system, so hopefully it isn't too hard to start adding lint rules like this as they come up!